ports/zephyr/modbluetooth_zephyr.c

```diff
@@ -404,6 +809,131 @@ int mp_bluetooth_gap_peripheral_connect_cancel(void) {
 
 #endif // MICROPY_PY_BLUETOOTH_ENABLE_CENTRAL_MODE
 
+// Note: modbluetooth UUIDs store their data in LE.
+static struct bt_uuid *create_zephyr_uuid(const mp_obj_bluetooth_uuid_t *uuid) {
+    struct bt_uuid *result = (struct bt_uuid *)m_new(union uuid_u, 1);
+    if (uuid->type == MP_BLUETOOTH_UUID_TYPE_16) {
+        bt_uuid_create(result, uuid->data, 2);
+    } else if (uuid->type == MP_BLUETOOTH_UUID_TYPE_32) {
+        bt_uuid_create(result, uuid->data, 4);
+    } else {    //  MP_BLUETOOTH_UUID_TYPE_128
+        bt_uuid_create(result, uuid->data, 16);
+    }
+    return result;
+}
+
+static void gatt_db_add(const struct bt_gatt_attr *pattern, struct bt_gatt_attr *attr, size_t user_data_len) {
+
```

Comment:
Please don't add blank lines at the start of functions.

Reply:
OK